### ISSUE TITLE
unicode proposal in RTF ballot

### ISSUE THREAD

**Issue by opatrascoiu** [dmn-tck/tck#319]
There is a proposal for the unicode issue https://issues.omg.org/issues/DMN13-127 going into the ballot next week.

The proposal is here https://issues.omg.org/issues/DMN13-165.

We raised two more in this space:
https://issues.omg.org/issues/DMN13-142
https://issues.omg.org/issues/DMN13-142

The ballot opens next week so we have time to review.

Lets discuss in the meeting.

**Comment by agilepro** [comment]
Link for 127 works, but 165 is not found.  Is that the right link?     Also, 142 works, but both links there are the same.

three things changed on 18th,   142, 127, and 143, 165.   Seems that 165 is there in some modes but does not display with an issues link.  Looks like lowercase u will be a four digit code point, while a capital U will be a 6 digit code point value.

**Comment by agilepro** [comment]
Thank Octavian for posting this.   We managed to get the links to work and reviewed this in the meeting.  It seems that the RTF decided to go in a completely different direction than we suggested.  That is their prerogative.  

It seems that there is no more issue for us to track here.   When release 1.3 of the spec comes out, all the vendors will have to deal with the change at that time, but until then there is really nothing we can do here.   So closing this issue.
